langchain/tools/playwright/click.py

```diff
@@ -26,6 +26,18 @@ class ClickTool(BaseBrowserTool):
     description: str = "Click on an element with the given CSS selector"
     args_schema: Type[BaseModel] = ClickToolInput
 
+    visible_only: bool = True
+    """Whether to consider only visible elements."""
+    strict_mode: bool = False
+    """Whether to employ Playwright's strict mode when clicking on elements."""
+    timeout: float = 1_000
```

Comment:
What are the units of this? Could you add to the description?

Reply:
personally don't feel need for prefix (all of this lives within playwright module after all), but don't feel strongly either way

Reply:
Updated with units for timeout and I did end up adding the prefixes to signify those values are passed to the Playwright engine.